lib/log.js

```diff
@@ -1,21 +1,37 @@
-var log4js = require('log4js')
-  , that;
+var util = require('util');
+var winston = require('winston');
 
-that = {
-  category: null,
-  level: 'INFO',
+module.exports = {
   init: function(config) {
-    config = config || {};
-    log4js.configure(config.log4js);
-    that.category = config.category || that.category;
-    that.level = config.level || that.level;
+    this.config = config || {};
   },
-  getLogger: function(category) {
-    var logger;
-    logger = log4js.getLogger(category || that.category);
-    logger.setLevel(that.level);
-    return logger;
+
+  getLogger: function() {
+    if (this.config.logger) {
+      return this.config.logger;
+    } else {
+      var logger = new winston.Logger({
+        transports: [
+          new winston.transports.Console({
+            level: this.config.level || 'warn',
+            formatter: function(options) {
+              return util.format(
+                '[%s] %s mongodb-fs %s',
```

Comment:
This wasn't really working before - run the test with the logging enabled (`LOG_LEVEL=trace mocha test/testInProcess.js`) and see all the modules use the same category. :smirk_cat: